app/src/main/java/protect/card_locker/importexport/StocardImporter.java

```diff
@@ -226,6 +228,18 @@ public ZIPData importZIP(ZipInputStream zipInputStream, final ZIPData zipData) t
                             ZipUtils.readJSON(zipInputStream)
                                     .getString("content")
                     );
+                } else if (fileName.endsWith("usage-statistics/content.json")) {
+                    JSONArray usages = ZipUtils.readJSON(zipInputStream).getJSONArray("usages");
+                    JSONObject lastUsedObject = usages.getJSONObject(usages.length() - 1);
```

Comment:
This code assumes the array is never empty. Which is hopefully correct. But maybe add a check just in case?

Reply:
I can add it when refactoring.